### PR TITLE
dhall-yaml 1.0.1 (new formula)

### PR DESCRIPTION
**Pull request by Gabriella439**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by bayandin** [comment]
Catalina:
```
Failed to install integer-logarithms-1.0.3
Build log ( /private/tmp/dhall-yaml-20200207-11755-19ri01t/dhall-yaml-1.0.1/.cabal-sandbox/logs/ghc-8.8.2/integer-logarithms-1.0.3-DB3mmMEnGUoGsmGCDooyTI.log ):
Downloading  mime-types-0.1.0.9
/private/tmp/dhall-yaml-20200207-11755-19ri01t/dhall-yaml-1.0.1/.cabal-sandbox/logs/ghc-8.8.2/integer-logarithms-1.0.3-DB3mmMEnGUoGsmGCDooyTI.log: openFile: does not exist (No such file or directory)
```

High Sierra:
```
Failed to install vector-0.12.1.2
Build log ( /private/tmp/dhall-yaml-20200207-83796-1s8iihq/dhall-yaml-1.0.1/.cabal-sandbox/logs/ghc-8.8.2/vector-0.12.1.2-9VEK1w6XuCFF3hLj52eD2N.log ):
Downloading  scientific-0.3.6.2
/private/tmp/dhall-yaml-20200207-83796-1s8iihq/dhall-yaml-1.0.1/.cabal-sandbox/logs/ghc-8.8.2/vector-0.12.1.2-9VEK1w6XuCFF3hLj52eD2N.log: openFile: does not exist (No such file or directory)
```

**Comment by athas** [comment]
Those look like transient file system errors in the CI system.  I've seen that before for Haskell packages.  It may go away if the build is just restarted.

**Comment by iMichka** [comment]
@BrewTestBot test this please

File: Formula/dhall-yaml.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  depends_on "cabal-install" => :build
+  depends_on "ghc" => :build
+
+  def install
+    install_cabal_package
+  end
+
+  test do
+    assert_match "1", pipe_output("#{bin}/dhall-to-yaml-ng", "1", 0)
```
Comment (SMillerDev) [inline Formula/dhall-yaml.rb]: Can you make the test a little more extensive?

**Comment by issyl0** [comment]
@BrewTestBot test this please

**Comment by issyl0** [comment]
Thank you for this, @Gabriel439! Without people like you submitting PRs, we couldn't run this project! :muscle:

**Pull request closed without merging**